easybuild/easyconfigs/b/bat/bat-0.3.3-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'Bundle'
+
+name = 'bat'
+version = '0.3.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/bat'
+description = """The BAT Python package supports the processing and analysis of Bro data with Pandas,
+scikit-learn, and Spark"""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('PyYAML', '3.12', versionsuffix),
+    ('scikit-learn', '0.19.1', versionsuffix),
+    ('arrow', '0.7.1', versionsuffix),
+    ('Spark', '2.2.0', '-Hadoop-2.6-Java-1.8.0_152%s' % versionsuffix),
+]
+
+exts_list = [
+    ('requests', '2.18.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/requests/'],
+        'checksums': ['9c443e7324ba5b85070c4a818ade28bfabedf16ea10206da1132edaa6dda237e'],
+    }),
+    ('pathtools', '0.1.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pathtools'],
+        'checksums': [
+            '7c35c5421a39bb82e58018febd90e3b6e5db34c5443aaaf742b3f33d4655f1c0',  # pathtools-0.1.2.tar.gz
```

Comment:
let's collapse this to a single line (`--inject-checksums` in `develop` should do so automagically)